storage/innobase/xtrabackup/src/keyring_components.cc

```diff
@@ -37,6 +37,8 @@ const char *XTRABACKUP_KEYRING_KMIP_CONFIG = "component_keyring_kmip.cnf";
 
 const char *XTRABACKUP_KEYRING_KMS_CONFIG = "component_keyring_kms.cnf";
 
+const char *XTRABACKUP_KEYRING_VAULT_CONFIG = "component_keyring_vault.cnf";
```

Comment:
a side track question: what about other components? Did PS implement them and we tested those?

Reply:
only manual testing too. There is no PS 8.1 for the bootstrap script to download from.

Reply:
manual testing = PXB MTR ran locally

Reply:
I mean, did we cherry pick commits from PS for kms and kmip? similar to this work

Reply:
Yes, those are part of 8.0 and were part of the initial 8.1 as part of the base branch.

Reply:
Thanks